### Geofrenzy GfDwellThreshold message
#
# A dwell message specifies a current entitlement in relation to the tracked
# location.
#
# The threshold dwell ... 
#
Header          header      # Standard ROS header
GfEntThreshold  entitlement # Dwell entitlement
